imod/msw/model.py

```diff
@@ -282,3 +305,66 @@ def regrid_like(
             regridded_model[mod2svat_name] = CouplerMapping()
 
         return regridded_model
+
+    @classmethod
+    def from_imod5_data(
+        cls,
+        imod5_data: Imod5DataDict,
+        target_dis: StructuredDiscretization,
+        times: list[datetime],
+    ):
+        """
+        Construct a MetaSWAP model from iMOD5 data in the CAP package, loaded
+        with the :func:`imod.formats.prj.open_projectfile_data` function.
+
+        Parameters
+        ----------
+        imod5_data: dict
+            Dictionary with iMOD5 data. This can be constructed from the
+            :func:`imod.formats.prj.open_projectfile_data` method.
+        target_dis: imod.mf6.StructuredDiscretization
+            Target discretization, cells where MODLOW6 is inactive will be
+            inactive in MetaSWAP as well.
+        times: list[datetime]
+            List of datetimes, will be used to set the output control times.
+            Is also used to infer the starttime of the simulation.
+
+        Returns
+        -------
+        MetaSwapModel
+            MetaSWAP model imported from imod5 data.
+        """
+        extra_paths = imod5_data["extra"]["paths"]
+        path_to_parasim = find_in_file_list("para_sim.inp", extra_paths)
+        parasim_settings = read_para_sim(path_to_parasim)
```

Comment:
What if `path_to_parasim` is empty? Should this be put in an if statement?

Reply:
All iMOD5 usecases contain this file in the "EXTRA" section in the projectfile: Without it they couldn't run their MetaSWAP model with iMOD5. If the para_sim.inp is missing an error will be thrown: "could not find para_sim.inp in list of paths: ..."